flink-yarn-tests/src/test/java/org/apache/flink/yarn/YarnTestBase.java

```diff
@@ -204,6 +217,36 @@ public static void populateYarnSecureConfigurations(Configuration conf, String p
 		conf.set("hadoop.security.auth_to_local", "RULE:[1:$1] RULE:[2:$1]");
 	}
 
+	public static void populateHDFSSecureConfigurations(Configuration conf, String principal, String keytab)
```

Comment:
why do we need `kerberos` here for testing?

Reply:
Ah. I see now. I think this relates to @tillrohrmann 's [comment](https://github.com/apache/flink/pull/8303/files#r315075331). once we refactor the test ITBase. it should be ok to not test against kerberized DFS cluster.